_posts/2019-05-06-gis-wisdom.md

```diff
@@ -44,20 +44,20 @@ Learn more about the [ArcGIS versioning process](http://desktop.arcgis.com/en/ar
+In GIS, we build network datasets for navigation. These networks are based on connectivity. We use terms such as *edges* and *junctions*, but really we’re just establishing resource groups to which we grant connectivity to other sources.
 
-Likewise, building a personal network helps us determine who and what we allow ourselves to connect with.  A well-thought-out network not only tells us where we can connect but also where we should not. Taking the time to think about this connectivity ensures that we build a quality social network.
+Likewise, building a personal network helps us determine who and what we allow ourselves to connect with. A well-thought-out network not only tells us where we can connect but also where we should not. Taking the time to think about this connectivity ensures that we build a quality social network.
 
-Learn more about [creating an ArcGIS network datasets](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/creating-a-network-dataset.htm) and [how connectivity works](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/understanding-connectivity.htm).
+Learn more about [creating ArcGIS network datasets](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/creating-a-network-dataset.htm) and [how connectivity works](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/understanding-connectivity.htm).
 
 ### 3. Extract, Translate, and Load (ETL) 
 {: .text-left}
 
 `being adaptable`
 
-You have probably heard the phrase, 'build once, use many'.  When building data, we say this when we want the output to be adaptable and used for many purposes.  Take road centerlines for example.  This dataset should be designed with many uses in mind (such as routing, mapping, locating, asset management, etc.).  With a simple ‘extract, translate, and load’ (ETL), the data should meet the needs of many users.  
+You may have heard the phrase "build once, use many." When building data, we say this when we want the output to be adaptable and used for many purposes. Take road centerlines for example. This dataset should be designed with many uses in mind (such as routing, mapping, locating, asset management, etc.). With a simple "extract, translate, and load" (ETL), the data should meet the needs of many users.  
 
-We should learn to be adaptable as well.  Life is not stagnant, and the ability to change and adapt is beneficial.  Having a single-use in mind has the potential of putting us back at the drawing board, building again.
+We should learn to be adaptable as well. Life is not stagnant, and the ability to change and adapt is beneficial. Having a single use in mind has the potential to put us back at the drawing board, building again.
```

Comment:
Here with "Having a single use in mind has..." I'm wondering whether you can explain that just a little more? I think I kind of get what you mean, but when I first read it I thought, "single use for what?"

Reply:
i see what you're saying.  i'll craft something up and incorporate the modification into an upcoming PR i'm going to make to the repo.  Thanks!